BestBuyCRUD/SalesRepo.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using MySql.Data.MySqlClient;
+
+namespace BestBuyCRUD
+{
+    class SalesRepo
+    {
+        public string connectionString; // field
+
+        public SalesRepo(string conn)  
+        {
+            connectionString = conn;
+        }
+
+        public List<Sales> GetSales()
+        {
+            MySqlConnection connect = new MySqlConnection(connectionString);
+
+            using (connect)
+            {
+                connect.Open();
+                MySqlCommand cmd = connect.CreateCommand();
+                cmd.CommandText = "SELECT SalesID AS SID, ProductId AS PID, Quantity AS Q,"
+                    + "Price, Date, EmployeeID AS EID From Sales;";
+
+                MySqlDataReader reader = cmd.ExecuteReader();
+
+                List<Sales> sales = new List<Sales>();
+                while(reader.Read())
+                {
+                    Sales s = new Sales
+                    {
+                        SalesId = (int)reader["SID"],
+                        ProductId = (int)reader["PID"],
+                        Quantity = (int)reader["Q"],
+                        Price = (decimal)reader["Price"],
+                        Date = Convert.ToDateTime(reader["Date"]),
+                        EmployeeId = (int)reader["EID"]
+                    };
+                    sales.Add(s);
+                }
+                return sales;
+            }
+        }
+
+        public void CreateSales(int pID,int quantity,decimal price,DateTime date,int eID)
```

Comment:
For consistency, there should probably be spaces after the commas in this parameter list

Reply:
glad I learned a new trick using short key (ctr + k + d)! it fixed most of the inconsistency with spacing!